src/sonic-config-engine/tests/sample_output/py2/qos-dell9332.json

```diff
@@ -0,0 +1,1101 @@
+        }
+    },
+    "WRED_PROFILE": {
+        "AZURE_LOSSLESS" : {
+            "wred_green_enable"      : "true",
+            "wred_yellow_enable"     : "true",
+            "wred_red_enable"        : "true",
+            "ecn"                    : "ecn_all",
+            "green_max_threshold"    : "2097152",
+            "green_min_threshold"    : "1048576",
+            "yellow_max_threshold"   : "2097152",
+            "yellow_min_threshold"   : "1048576",
+            "red_max_threshold"      : "2097152",
+            "red_min_threshold"      : "1048576",
+            "green_drop_probability" : "5",
+            "yellow_drop_probability": "5",
+            "red_drop_probability"   : "5"
+        }
+    },
+    "QUEUE": {
+        "Ethernet0|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet8|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet16|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet24|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet32|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet40|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet48|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet56|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet64|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet72|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet80|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet88|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet96|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet104|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet112|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet120|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
+        },
+        "Ethernet128|3": {
+            "scheduler"   : "[scheduler.1]",
+            "wred_profile": "[AZURE_LOSSLESS]"
```

Comment:
remove references. should be 
"scheduler": "scheduler.1",
"wred_profile": "AZURE_LOSSLESS"